content/types/_http_paginated_response.textile

```diff
@@ -0,0 +1,17 @@
+An @HttpPaginatedResponse@ is a "@PaginatedResult@":#paginated-result that carries additional information about the response to an HTTP request. It is used when "making custom HTTP requests":/rest/usage#request.
```

Comment:
I think it's generally unwise to use an anchor link that is not a full path in a partial as you don't know that the other partial exists.